### PR TITLE
2023 April Updates:  - fqcn - linting and versions - warning method - typos

### PR DESCRIPTION
**Pull request by frederickw082922**
Update:
- fqcn
- linting and versions
- warning method
- typo fixes

### CODE REVIEW

File: tasks/cis_pgs12_redhat_fixes.yml [hunk 1]
```diff
@@ -146,24 +157,23 @@
...
       - rhel
       - rule_1.5
 
 # Section 2
 # Directory and File Permissions
-- name: "SCORED | 2.1 | PATCH | Ensure the file permissions mask is correct"
+- name: "2.1 | PATCH | Ensure the file permissions mask is correct"
   block:
-      - name: "SCORED | 2.1 | AUDIT | Ensure the file permissions mask is correct | Get user environment file (.bashrc, .bash_profile, or .profile}"
+      - name: "2.1 | PATCH | Ensure the file permissions mask is correct | Get user environment file (.bashrc, .bash_profile, or .profile}"
```
Comment (MrSteve81) [inline tasks/cis_pgs12_redhat_fixes.yml:168]: Change to Audit

**Review by MrSteve81 — COMMENTED** [review]
Take a look at comments

File: LICENSE
Comment (MrSteve81) [inline LICENSE:1]: remove Lockdown Enterprise Releases

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Ansible Lockdown
+Copyright (c) 2023 Mindpoint Group / Lockdown Enterprise / Lockdown Enterprise Releases
```
Comment (MrSteve81) [inline LICENSE:3]: remove Lockdown Enterprise Releases

File: tasks/cis_pgs12_redhat_fixes.yml (hunk 1 above)
Comment (frederickw082922) [reply tasks/cis_pgs12_redhat_fixes.yml:168]: Good catch! Thank you @MrSteve81 !

File: LICENSE (hunk 2 above)
Comment (frederickw082922) [reply LICENSE:3]: Thank you @MrSteve81 !

**Comment by frederickw082922** [comment]
Moved changes to new branch 2023_CIS_Restructure

**Pull request closed without merging**